cmd/minishift/cmd/start.go

```diff
@@ -537,7 +537,7 @@ func ocSupportFlag(cmdName string, flag string) bool {
 
 func setSubcriptionManagerParameters() {
 	cluster.RegistrationParameters.Username = viper.GetString(username)
-	cluster.RegistrationParameters.Password = viper.GetString(password)
+	cluster.RegistrationParameters.Password = util.EscapeSingleQuote(viper.GetString(password))
```

Comment:
IMO this should be done here - https://github.com/minishift/minishift/blob/088338ecc7071f56c869eb2ffe7ec78df722f734/pkg/minishift/registration/redhat.go#L60-L60. This is just needed for using the password as part of this command.